content/postgresql/concepts/aggregate-functions/aggregate-functions.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: 'Aggregate Functions'
+Description: 'Performs calculations across multiple rows of data, returning a single result.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'PostgreSQL'
+  - 'Aggregate Functions'
+CatalogContent:
+  - 'learn-postgresql'
+  - 'paths/analyze-data-with-sql'
+---
+
+In PostgreSQL, **aggregate functions** perform calculations across multiple rows of data and return a single result. These functions execute operations including counting, adding, finding the average, and searching the maximum or minimum value(s) using the `SELECT` statement and the `GROUP BY` clause.
```

Comment:
```suggestion
In PostgreSQL, **aggregate functions** perform calculations on a data set and return a single result. These functions include counting, adding, finding the average, and searching the maximum or minimum value(s) using the `SELECT` statement and the `GROUP BY` clause.
```

Reply:
Since it's not compulsory to use `GROUP BY` with Aggregate Functions mention that in practice we use `GROUP BY`